docs/tutorial.rst

```diff
@@ -1,11 +1,10 @@
 Tutorial
 ========
 
-In the examples below, output files are written to the
-``pyuvdata/pyuvdata/data/tutorial_output`` directory, which you will need to
-create to run the code exactly as written. Alternatively you can change the
-location the output files are saved to by changing the arguments to the
-``os.path.join`` calls.
+In the examples below, output files are written to a temporary directory created during testing.
+By default they will write to the current working directory.
+Alternatively you can change the location the output files are saved to
+by changing the arguments to the ``os.path.join`` calls.
```

Comment:
I think this is a little confusing because most of our users won't realize we run it in tests. Maybe lead with the part about what happens when they run the code themselves (it writes to the current working directory). Then optionally end with a sentence or parenthetical about what happens during testing.